### PR TITLE
Add IT Advisors Nepal and QPay

### PR DESCRIPTION
**Pull request by pfieffer**
* Add IT Advisors Nepal
* Add QPay

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Incessant Rain Animation Studios](http://www.incessantrain.com/) | Chhauni, Kathmandu | We specialize in 3D character animation, visual effects for feature films, game cinematics, commercials and integrated media
 - [Insight Workshop](https://insightworkshop.io/) | Kamalpokhari, Kathmandu | Software Development Company
 - [IntroCept Nepal](http://nepal.introcept.co/) | Dhobighat, Lalitpur | IntroCept is a concept development company with skills and expertise to facilitate complex business solutions
+- [IT Advisors Nepal](http://www.itadvisorsnepal.com/) | Dhumbarahi Kathmandu | IT Consulting, Application Architecture and Digital Solutions
```
Comment (mesaugat) [inline README.md]: Comma missing `Dhumbarahi, Kathmandu`

File: README.md [hunk 2]
```diff
@@ -98,6 +99,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Pioneer Solutions](http://www.pioneersolutionsglobal.com/) | Sitapaila Chowk, Kathmandu | Next Generation C/ETRM and EMIS Software
 - [Professional Computer System Pvt. Ltd.](http://www.pcs.com.np/) | Chakupat, Lalitpur | We develop national or enterprise-wide application for Government, Banking & Financial Sectors and Manufacturing Industries Verticals
 - [Proshore](http://proshore.nl/) | Tinkune, Kathmandu | New media solutions that enable you to achieve your goal
+- [QPay](http://www.qpay.com.np/) | Upahaar Marg- 35 , Baluwatar Kathmandu | QPay Pvt. Ltd. Incorporated and registered in Nepal has brought forward mobile based payment solution
```
Comment (mesaugat) [inline README.md]: We are only using the name of the place and the city. Can you make the change and only put `Baluwatar, Kathmandu` and push again.

File: README.md (hunk 1 above)
Comment (pfieffer) [reply README.md]: Oh yeah.

File: README.md (hunk 2 above)
Comment (pfieffer) [reply README.md]: Okay. I have made the changes.

**Pull request merged**